app/assets/stylesheets/generic/timeline.scss

```diff
@@ -70,7 +70,7 @@
           height: 0;
           border-style: solid;
           border-width: 9px 9px 9px 0;
-          border-color: transparent #f5f5f6 transparent transparent;
+          border-color: transparent $background-color transparent transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.